app/controllers/users_controller.rb

```diff
@@ -16,7 +16,22 @@ def create
     end
   end
 
-  def show
+  def edit
+    if User.find(params[:id]).admin?
```

Comment:
@meyerhoferc I originally used something similar, but I wanted to make sure that if an admin manually plugged in a path to another user's edit page (like 'users/2/edit') they saw a 404. I realize now though that they could edit another admin's info if we keep it this way. Good catch. 👍